src/frontend/src/pages/Groups/GroupsDataTable.jsx

```diff
@@ -23,15 +25,26 @@ export function GroupsDataTable({ groups }) {
         { key: 'memberCount', header: 'Members' }
     ];
 
+    const navigate = useNavigate();
+
     const getRow = rowId => groups.find(({ id }) => id == rowId);
 
     return (
         <SmsDataTable rows={groups} headers={headers} className='groups-page__datatable'>
             {({ rows, headers, getHeaderProps, getRowProps, getTableProps, onInputChange }) => (
-                <TableContainer title='Groups'>
+                <TableContainer 
+                    title='Groups'
+                    description='This is a list of all the project for this semester.'
+                >
                     <TableToolbar>
                         <TableToolbarContent>
                             <TableToolbarSearch defaultExpanded={true} onChange={onInputChange} />
+                            <Button 
+                                onClick={() => navigate(`${AppRoutes.groups}/add`)}
+                                size="small" 
```

Comment:
this is an invalid prop, and the table defines a fixed size already. Deleting for now,

```suggestion
```